django/conf/__init__.py

```diff
@@ -25,6 +27,10 @@
     'The FILE_CHARSET setting is deprecated. Starting with Django 3.1, all '
     'files read from disk must be UTF-8 encoded.'
 )
+PASSWORD_RESET_TIMEOUT_DAYS_DEPRECATED_MSG = (
+    'The PASSWORD_RESET_TIMEOUT_DAYS setting is deprecated. Starting with '
+    'Django 4.0, use PASSWORD_RESET_TIMEOUT instead.'
```

Comment:
`The PASSWORD_RESET_TIMEOUT_DAYS setting is deprecated. Use PASSWORD_RESET_TIMEOUT instead.`